patches/minecraft/net/minecraft/server/Main.java.patch

```diff
@@ -101,8 +101,7 @@
 +               // FORGE: Halting as GameTestServer will cause issues as it always calls System#exit on both crash and normal exit, so skip it
 +               if (!(dedicatedserver instanceof net.minecraft.gametest.framework.GameTestServer))
                 dedicatedserver.m_7570_(true);
-+               //TODO-PATCHING: Figure out what needs to be done here after the logging update.
-+               //LogManager.shutdown(); // we're manually managing the logging shutdown on the server. Make sure we do it here at the end.
++               org.apache.logging.log4j.LogManager.shutdown(); // we're manually managing the logging shutdown on the server. Make sure we do it here at the end.
```

Comment:
Mojang use SLF4J now. We should see if there's a way to properly decouple this from hardcoded Log4j paths.

Reply:
I believe this decoupling should be left to another PR and this just restore previous behavior that still works *at the moment*. There are still many usages of LogManager across the Forge codebase and the use of log4j markers instead of slf4j meaning not everything can easily be moved to LogUtils.

Reply:
I agree with @SizableShrimp here. SLF4J is only a facade, and needs a backing provider to properly function. Currently, Log4j is that backing provider. Until such time that Mojang replaces Log4j with another logging framework (like logback) and Forge also moves away from Log4j to SLF4J, we should keep (or in this case, return) this shutdown call to Log4j's `LogManager`.

(Moving Forge away from Log4j to SLF4J should be done at another time, not in this PR.)